### PR TITLE
S1 one 160 Home page layout and styles partially applied

### PR DESCRIPTION
**Pull request by evasquez26**
## Story / Task

[ONE-160](https://bixlabs.atlassian.net/browse/ONE-160)

## What this PR does?

Adds layout and styles for home components. Adds the right panel with the social links on the footer.

### Before 

Styles on the navbar, icons and right panel was missing.

### After

- Added styles to main page
- Added a right panel which will contain the social links

### How to test / reproduce

- Run the app
- Observe that most of the styles have been applied to the site

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO

Thanks!

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
     <div class="col-xs-7">
       <router-outlet></router-outlet>
     </div>
-    <div class="col-xs-3"></div>
+    <div class="col-xs-3">
+      <right-panel></right-panel>
```
Comment (evasquez26) [inline src/app/app.component.html]: This needs to be changed, using the method of #11

File: src/app/contactus/contactus.component.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@import '../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables';
```
Comment (iwannabaa) [inline src/app/contactus/contactus.component.scss]: We can remove `../../../` here

File: src/app/header/header.component.html [hunk 3]
```diff
@@ -1,64 +1,79 @@
...
+              <a [routerLink]="['/notification-list']" href="#">
+                <p class="text-primary text-center">View all</p>
+              </a>
+            </li>
+          </ul>
+        </li>
+        <!-- Notifications -->
+        <li class="user-nav-link">
+          <img class="profile-img" alt="Profile" src="assets/img/profile-img.png">
+          <h5 class="user-info">
```
Comment (iwannabaa) [inline src/app/header/header.component.html]: What if we show the `Welcome, User` only if we have the user info, that way if no user info is retrieved, we display the avatar only.

File: src/styles.scss [hunk 4]
```diff
@@ -6,11 +6,148 @@
...
+  text-transform: none;
+  line-height: 34px;
+  font-size: 26px;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-linkedin-square {
```
Comment (iwannabaa) [inline src/styles.scss]: In commit `ea48e1bb9184aeab25d300f313b8970b4e6ef975` I added the fonts.css as an import :)

File: src/styles/_variables.scss [hunk 5]
```diff
@@ -6,8 +6,33 @@ $icon-font-path: '.~bootstrap-sass/assets/fonts/bootstrap/';
...
 /* --------------------------------------- */
 /* APP VARIABLES                         */
 /* --------------------------------------- */
+
+$accent-color: #73c261;
+$default-grey-color: #455a64;
+
+$icomoon-font-path: "assets/fonts" !default;
+
+$icon-linkedin-square: "\e902";
```
Comment (iwannabaa) [inline src/styles/_variables.scss]: As I added the fonts.css we don't need to add these variables for `content`

File: src/app/header/header.component.html (hunk 3 above)
Comment (evasquez26) [inline src/app/header/header.component.html]: Well in theory, every user that registers must provide his name and last name so I don't know if that scenario is ever going to happen

File: src/styles/_variables.scss (hunk 5 above)
Comment (evasquez26) [inline src/styles/_variables.scss]: Thank you!

File: src/styles.scss (hunk 4 above)
Comment (evasquez26) [inline src/styles.scss]: got it, thanks!

**Pull request merged**